build_debian.sh

```diff
@@ -678,6 +678,7 @@ sudo rm -f $ONIE_INSTALLER_PAYLOAD $FILESYSTEM_SQUASHFS
 ## Note: -x to skip directories on different file systems, such as /proc
 sudo du -hsx $FILESYSTEM_ROOT
 sudo mkdir -p $FILESYSTEM_ROOT/var/lib/docker
+sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c "truncate -s 0 /etc/resolv.conf"
```

Comment:
Some users might intend to add some entries to `/etc/resolv.conf` for final build image.
This truncate might remove them unexpected. Maybe have a resolv.conf in `/files/image_config/resolv.conf` would be a better solution? We can override the file by using `/files/image_config/resolv.conf` instead truncate the file. And user can modify the image_config if they intent to have some entries in final image.